Update README.md (#375)
OctoMind-dev/debugtopus: README.md

# debugtopus  🐙 💻

> Hey! We use AI agent to auto-generate, host and run AI-generated end-to-end tests in standard Playwright code in the cloud. We work on both staging and production, as long as we can access the website. 

Author: ```suggestion
Hey!  Debugtopus is a tool to run Octomind tests against the local instance of your app. Octomind tests are AI auto-generated, hosted and run end-to-end tests in standard Playwright code in the cloud. They work on both staging and production, as long as we can access the website. 
```

> Whether you want to make sure you didn't break a new build or use the e2e tests for synthetic monitoring, we offer a fast and easy way to QA your web app. Here some [info](https://www.octomind.dev/?utm_source=github&utm_medium=debugtopus&utm_campaign=rm) on that and more in the [docs](https://www.octomind.dev/docs?utm_source=github&utm_medium=debugtopus&utm_campaign=rm).

Author: ```suggestion
Here some [info](https://www.octomind.dev/?utm_source=github&utm_medium=debugtopus&utm_campaign=rm) on that and more in the [docs](https://www.octomind.dev/docs?utm_source=github&utm_medium=debugtopus&utm_campaign=rm).
```

> We created Debugtopus to help you pinpoint the bug if a test goes red. It's a tool to run Octomind tests against a local instance of your app.

Author: ```suggestion
```

# debugtopus  🐙 💻

> Hey!  Debugtopus is a tool to run Octomind tests against the local instance of your app. Octomind tests are AI auto-generated, hosted and run end-to-end tests in standard Playwright code in the cloud. They work on both staging and production, as long as we can access the website. 

Reviewer: ```suggestion
Debugtopus is a tool to run Octomind tests against the local instance of your app. Octomind tests are AI auto-generated, hosted and run end-to-end tests in standard Playwright code in the cloud. 
```

> Here some [info](https://www.octomind.dev/?utm_source=github&utm_medium=debugtopus&utm_campaign=rm) on that and more in the [docs](https://www.octomind.dev/docs?utm_source=github&utm_medium=debugtopus&utm_campaign=rm).

Reviewer: ```suggestion
Here's some more [info](https://www.octomind.dev/?utm_source=github&utm_medium=debugtopus&utm_campaign=rm) on octomind and more in the [docs](https://www.octomind.dev/docs?utm_source=github&utm_medium=debugtopus&utm_campaign=rm).
```

Reviewer: Im not sure if UTM-links in readme are good style, devs are finicky like that. But also fine I guess if you need it.
